c7n/tags.py

```diff
@@ -903,7 +903,7 @@ class UniversalTagDelayedAction(TagDelayedAction):
 
     batch_size = 20
     concurrency = 2
-    permissions = ('resourcegroupstaggingapi:TagResources',)
+    permissions = ('tag:TagResources',)
```

Comment:
just remove the line, its not used. all permissions annotations that are used are validated per #5242